solutions/add_two_numbers.ts

```diff
@@ -0,0 +1,17 @@
+import { SinglyLinkedListNode } from '../types/linked_list.ts';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/axross/leetcode-typescript/pullRequest?prid=7152801)
```suggestion
import { SinglyLinkedListNode } from "../types/linked_list.ts";
```